<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="qMRMLWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="ctkCollapsibleButton" name="parameterSetSelectionCollapsibleButton">
     <property name="text">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Parameters: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="parameterNodeSelector">
        <property name="toolTip">
         <string>Pick parameter node</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScriptedModuleNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="baseName">
         <string>FiducialsToModelDistance</string>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkCollapsibleButton" name="inputsOutputsCollapsibleButton">
     <property name="text">
      <string>Fiducial to fiducial/model distance</string>
     </property>
     <property name="collapsedHeight">
      <number>150</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input points:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input reference points or surface:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Output error metrics table:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="outputMetricsTableNodeSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTableNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Error metrics</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Output point distances table:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="outputDistancesTableNodeSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTableNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Point distances</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="showMetricsTablePushButton">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="showDistancesTablePushButton">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="inputPointsSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="inputReferenceNodeSelector">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QPushButton" name="applyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Run the algorithm.</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>UtilTest</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>parameterNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UtilTest</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputPointsSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UtilTest</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputReferenceNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UtilTest</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputMetricsTableNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UtilTest</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputDistancesTableNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputMetricsTableNodeSelector</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>showMetricsTablePushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputDistancesTableNodeSelector</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>showDistancesTablePushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
